packages/subgraph/src/packages/admin/admin.ts

```diff
@@ -117,12 +120,11 @@ export function handleProposalExecuted(event: ProposalExecuted): void {
 export function handleMembershipContractAnnounced(
   event: MembershipContractAnnounced
 ): void {
-  let dao = event.params.definingContract;
-  let packageEntity = AdminstratorAdminPlugin.load(event.address.toHexString());
-
-  if (packageEntity) {
-    // TODO: Query `dao` for addresses holding the `EXECUTE_PROPOSAL_PERMISSION` on `where = event.address` and store them as `administrators`
-    // <MISSING_IMPLEMENTATION>
-    // packageEntity.save();
-  }
+  let context = new DataSourceContext();
+  context.setString('pluginAddress', event.address.toHexString());
+  context.setString(
+    'permissionId',
+    '0xf281525e53675515a6ba7cc7bea8a81e649b3608423ee2d73be1752cea887889'
```

Comment:
can't we use `crypto.keccak256` ? or if not, the comment would be good that it's a hash of EXECUTE_PERMISSION

Reply:
same as above. I don't want to impact the indexing speed too much